### ISSUE TITLE
FramePack training details

### ISSUE THREAD

**Issue by cmmclee** [lllyasviel/FramePack#567]
Dear Authors,

I hope this message finds you well. I’m exploring FramePack training based on either Hunyuan or Wan and would appreciate your guidance on the following:

1、Base Model Architecture:
Is the base model designed for Text-to-Video (T2V) or Image-to-Video (I2V) tasks?

2、Parameter Fine-Tuning Strategy:
When freezing most parameters, is it sufficient to fine-tune only the PatchEmbedForCleanLatents modules (proj, proj_2x, proj_4x)?
Or would you recommend fine-tuning all parameters or selectively unfreezing the first few DiT blocks for better performance?

3、Training Convergence:
Approximately how many training steps are typically needed for convergence under standard settings (e.g., dataset size, batch size)?

Thank you for your time and insights! Looking forward to your response.

**Comment by Auswolf-debug** [comment]
1) It can do both.
2) look at the pick below and see the results.
3) The test i ran consisted of 12 videos @ 4 sec 30fps, batch size 1, Training resolution 256 x 256.

So this was my first run, I had already started training before i realized, i needed to adjust "Max video frames per sample" .
It was at 36 should of adjusted it to 120, to suit the data set.(seconds x fps).

Training time was 20 minutes on a rtx5060ti, honestly i thought nothing would happen due to the mistake above. Also to be noted my prompts suck in my dataset lol.

first epoch
Epoch: 1/50 | Step: 2/9 | Loss: 1.5874 | Avg Epoch Loss: 1.6185 | Speed: 0.38 it/s | Elapsed: 6s | ETA: 23m 26s

last epoch
Epoch: 50/50 | Step: 9/9 | Loss: 0.0279 | Avg Epoch Loss: 0.0636 | Speed: 0.38 it/s | Elapsed: 19m 49s | ETA: 2s

Picture of settings used for training.

![Image](https://github.com/user-attachments/assets/859f2f58-95cd-4813-b48a-df9d568f0dda)

And the results, 4 sec video.
Prompt = A women stands up and walks away.
No lora loaded.

https://github.com/user-attachments/assets/09f50391-c727-4021-97f1-77c831c0cb5d

Lora loaded.

https://github.com/user-attachments/assets/74461573-6c0d-4253-9ddc-4d419ad3a5aa


So if anyone has tips on settings please feel free to speak up. And before anyone asks, i will be sharing this trainer once its finished....

**Comment by julia-0105** [comment]
> 1. It can do both.它两者都能做。
> 2. look at the pick below and see the results.看看下面的截图，看看结果。
> 3. The test i ran consisted of 12 videos @ 4 sec 30fps, batch size 1, Training resolution 256 x 256.我进行的测试包括 12 个视频，每个视频 4 秒，30fps，批量大小 1，训练分辨率 256 x 256。
> 
> So this was my first run, I had already started training before i realized, i needed to adjust "Max video frames per sample" .所以这是我第一次运行，在我意识到需要调整"每个样本的最大视频帧数"之前，我已经开始了训练。 It was at 36 should of adjusted it to 120, to suit the data set.(seconds x fps).它应该是 36，应该调整为 120，以适应数据集（秒数 x 帧率）。
> 
> Training time was 20 minutes on a rtx5060ti, honestly i thought nothing would happen due to the mistake above. Also to be noted my prompts suck in my dataset lol.在 RTX 5060 Ti 上训练时间为 20 分钟，老实说，由于上述错误，我以为什么都不会发生。还要注意的是，我的数据集中的提示很差，哈哈。
> 
> first epoch  第一轮 Epoch: 1/50 | Step: 2/9 | Loss: 1.5874 | Avg Epoch Loss: 1.6185 | Speed: 0.38 it/s | Elapsed: 6s | ETA: 23m 26s轮次: 1/50 | 步骤: 2/9 | 损失: 1.5874 | 平均轮次损失: 1.6185 | 速度: 0.38 it/s | 已用时间: 6s | 预计剩余时间: 23m 26s
> 
> last epoch  最后一轮 Epoch: 50/50 | Step: 9/9 | Loss: 0.0279 | Avg Epoch Loss: 0.0636 | Speed: 0.38 it/s | Elapsed: 19m 49s | ETA: 2s轮次: 50/50 | 步骤: 9/9 | 损失: 0.0279 | 平均轮次损失: 0.0636 | 速度: 0.38 it/s | 已用时间: 19m 49s | 预计剩余时间: 2s
> 
> Picture of settings used for training.用于训练的设置图片。
> 
> ![Image](https://github.com/user-attachments/assets/859f2f58-95cd-4813-b48a-df9d568f0dda)
> 
> And the results, 4 sec video.以及结果，4 秒视频。 Prompt = A women stands up and walks away.提示 = 一位女士站起来走开。 No lora loaded.  没有加载 lora。
> 
>  250524_101742_827_4634_28.mp4 
> Lora loaded.  加载了 lora。
> 
>  250524_102712_689_3919_28.mp4 
> So if anyone has tips on settings please feel free to speak up. And before anyone asks, i will be sharing this trainer once its finished....如果有任何关于设置的技巧，请随时分享。至于有人可能会问，我会在训练器完成后分享它……

hello, I want to know what is your base model? wan2.1-14B or hunyuan-14B?t2v or i2v>

**Comment by Auswolf-debug** [comment]
Base model is framepack nf4.

**Comment by julia-0105** [comment]
Hi there, If I understand correctly, you're saying that the base model was not switched from HunYuanVideo to Wan, is that right?I’ve just followed you — great work! I’m really impressed by your Gradio interface for the training. If it's convenient for you, would you mind sharing the training script? Thank you!

> Base model is framepack nf4.基础模型是 framepack nf4。

**Comment by julia-0105** [comment]
> Dear Authors,  尊敬的作者们，
> 
> I hope this message finds you well. I’m exploring FramePack training based on either Hunyuan or Wan and would appreciate your guidance on the following:我希望这封信息能找到你。我正在探索基于 Hunyuan 或 Wan 的 FramePack 训练，并希望在你以下方面的指导：
> 
> 1、Base Model Architecture:1、基础模型架构： Is the base model designed for Text-to-Video (T2V) or Image-to-Video (I2V) tasks?基础模型是为文本到视频（T2V）任务还是图像到视频（I2V）任务设计的？
> 
> 2、Parameter Fine-Tuning Strategy:2、参数微调策略： When freezing most parameters, is it sufficient to fine-tune only the PatchEmbedForCleanLatents modules (proj, proj_2x, proj_4x)?当冻结大多数参数时，是否只需微调 PatchEmbedForCleanLatents 模块（proj、proj_2x、proj_4x）就足够了？ Or would you recommend fine-tuning all parameters or selectively unfreezing the first few DiT blocks for better performance?或者，您建议微调所有参数，还是选择性地解冻前几个 DiT 模块以获得更好的性能？
> 
> 3、Training Convergence:  3、训练收敛： Approximately how many training steps are typically needed for convergence under standard settings (e.g., dataset size, batch size)?在标准设置下（例如数据集大小、批处理大小），通常需要多少训练步骤才能收敛？
> 
> Thank you for your time and insights! Looking forward to your response.感谢您的时间和见解！期待您的回复。
Hello, I hope you're doing well. I'd like to check in and ask if there has been any update or progress recently? I'm planning to perform training with Wan, but it seems most of the current work is based on HunYuan.

**Comment by nankepan** [comment]
> Base model is framepack nf4.

Hello, I am currently attempting to reproduce the frampack training process. Would you be able to share the training code? I would greatly appreciate your help. Thank you!

**Comment by KaidDuong** [comment]
> > Base model is framepack nf4.
> 
> Hello, I am currently attempting to reproduce the frampack training process. Would you be able to share the training code? I would greatly appreciate your help. Thank you!

[musubi-tuner](https://github.com/kohya-ss/musubi-tuner)  This repository provides scripts for training LoRA (Low-Rank Adaptation) models with **HunyuanVideo, Wan2.1 and FramePack architectures.**
